node/core/av-store/src/tests.rs

```diff
@@ -16,11 +16,14 @@
 
 use super::*;
 
+use std::convert::TryFrom;
```

Comment:
Since there's no use of this trait anywhere (it seems), I believe it may be incorrect to implement the `TryFrom` trait as oppose to just making a Method on `Proof`.

Reply:
How would it be incorrect?